app/enterprise/2.3.x/plugins/oidc-curity.md

```diff
@@ -0,0 +1,209 @@
+
+### Curity Setup
+
+The Kong Dev Portal needs a client configured in the Curity Identity Server. The [Curity Getting Started Guide][curity-getting-started] outlines details on how to configure a client. The configuration details below should be sufficient:
+
+- Authentication method: `secret`. Make note of the Secret and use it in the `client_secret` field in the Developer Portal configuration (see below).
+- Capabilities: `Code Flow`.
+- Following scopes: `openid`, `profile`, and `email`.
+- Choose suitable authentication methods.
+- Add redirect URI (by default `http://kong:8004/<WORKSPACE_NAME>/auth`).
+
+NOTE: There is an issue with short-lived Access Tokens that is under investigation. Increase the `Access Token Time to Live` in the client configuration to `3000` as a temporary workaround. 
+
+More information is available in the [Code Flow][curity-code-flow-tutorial] tutorial.
+
+### Configure OpenID Connect in the Kong Developer Portal
+
+Enabling the Kong Developer Portal is outlined in the [Kong Dev Portal Documentation][kong-dev-portal-doc] and not covered in this article. The documentation also covers how to configure the [OpenID Connect Plugin][kong-dev-portal-doc-oidc].
+
+#### Example Configuration Object
+
+Below is an example configuration object that is used to configure the OIDC plugin for the Dev Portal.
+
+```json
+{
+    "redirect_uri": ["https://kong-dev-portal:8004/default/auth"],
+    "consumer_by": ["username","custom_id","id"],
+    "leeway": 1000,
+    "scopes": ["openid","profile","email"],
+    "logout_query_arg": "logout",
+    "login_redirect_uri": ["https://kong-dev-portal:8003"],
+    "login_action": "redirect",
+    "logout_redirect_uri": ["https://kong-dev-portal:8003"],
+    "ssl_verify": false,
+    "client_id": ["kong-dev-portal-client"],
+    "forbidden_redirect_uri": ["https://kong-dev-portal:8003/unauthorized"],
+    "client_secret": ["Pa$$w0rd!"],
+    "issuer": "https://idsvr.example.com/oauth/v2/oauth-anonymous/",
+    "logout_methods": ["GET"],
+    "consumer_claim": ["email"],
+    "login_redirect_mode": "query"
+}
+```
+
+![Enable OIDC in Kong Dev Portal](/assets/images/docs/dev-portal/curity/kong-dev-portal.png)
+
+### Curity Authentication Action
+
+An Authentication Action to automatically provision the user to the Kong Developer Portal is available in the Curity GitHub repository. Using the Action is not mandatory as the user could be provisioned in other ways, such as manually through the Kong Developer portal login screen. However, using the Authentication Action would streamline the user flow as the Action will take the user's full name and the email from the Curity Authenticator and automatically provision that to the Kong Developer Portal using the exposed API.
```

Comment:
```suggestion
An Authentication Action to automatically provision the user to the Kong Developer Portal is available in the Curity GitHub repository. Using the Action is not mandatory as the user could be provisioned in other ways, such as manually through the Kong Developer portal login psge. However, using the Authentication Action would streamline the user flow since the Action takes the user's full name and the email from the Curity Authenticator and automatically provision that to the Kong Developer Portal using the exposed API.
```